marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/endpoints/collectorLib.sjs

```diff
@@ -54,7 +61,14 @@ class CollectorLib {
       return sourceQuery;
     }
 
-    return `cts.uris(null, null, ${sourceQuery})`;
+    let query;
```

Comment:
For brevity, can use ? here - e.g. return sourceQueryLimit ? the first query : the second query